src/panel/pages/timeline/components/TimelineEvent.tsx

```diff
@@ -1,15 +1,60 @@
-import React, { FC, useContext, useMemo } from "react";
-import styled, { ThemeContext } from "styled-components";
+import React, { FC, useMemo } from "react";
+import styled, {
+  css,
+  FlattenSimpleInterpolation,
+  FlattenInterpolation,
+  ThemeProps,
+  DefaultTheme
+} from "styled-components";
 import { ReceivedDebugEvent } from "../../../types";
 import { useTooltip, TimelineTooltip } from "./TimelineTooltip";
 
-const EventDot = styled.div`
-  background: ${props => props.color};
-  border-radius: 50%;
-  border: solid 2px ${props => props.theme.dark["+1"]};
+const EVENT_SIZE = "10px";
+const shapeMap: Record<
```

Comment:
I'm not totally sure what this does - is this creating the shapes for the events? Does it help/make our lives easier if I send over the SVGs?

Reply:
It is yup - SVG's would work as well if we didn't fancy keeping it in just css